rocketpy/prints/flight_prints.py

```diff
@@ -356,40 +356,35 @@ def maximum_values(self):
             )
         )
         print(
-            "Maximum Acceleration: {:.3f} m/s² at {:.2f} s".format(
-                self.flight.max_acceleration, self.flight.max_acceleration_time
+            "Maximum Acceleration During Motor Burn: {:.3f} m/s² at {:.2f} s".format(
+                self.flight.max_acceleration_power_on,
+                self.flight.max_acceleration_power_on_time,
             )
         )
         print(
-            "Maximum Gs: {:.3f} g at {:.2f} s".format(
-                self.flight.max_acceleration
+            "Maximum Gs During Motor Burn: {:.3f} g at {:.2f} s".format(
+                self.flight.max_acceleration_power_on
                 / self.flight.env.gravity(
-                    self.flight.z(self.flight.max_acceleration_time)
+                    self.flight.z(self.flight.max_acceleration_power_on_time)
                 ),
-                self.flight.max_acceleration_time,
+                self.flight.max_acceleration_power_on_time,
             )
         )
-        # check if max acceleration during burn is different than overall
-        # max acceleration
-        if (
-            self.flight.max_acceleration_time
-            != self.flight.max_acceleration_power_on_time
-        ):
-            print(
-                "Maximum Acceleration During Motor Burn: {:.3f} m/s² at {:.2f} s".format(
-                    self.flight.max_acceleration_power_on,
-                    self.flight.max_acceleration_power_on_time,
-                )
+        print(
+            "Maximum Acceleration After Motor Burn: {:.3f} m/s² at {:.2f} s".format(
+                self.flight.max_acceleration_power_off,
+                self.flight.max_acceleration_power_off_time,
             )
-            print(
-                "Maximum Gs During Motor Burn: {:.3f} g at {:.2f} s".format(
-                    self.flight.max_acceleration_power_on
-                    / self.flight.env.gravity(
-                        self.flight.z(self.flight.max_acceleration_power_on_time)
-                    ),
-                    self.flight.max_acceleration_power_on_time,
-                )
+        )
+        print(
+            "Maximum Gs After Motor Burn: {:.3f} g at {:.2f} s".format(
+                self.flight.max_acceleration_power_off
+                / self.flight.env.gravity(
+                    self.flight.z(self.flight.max_acceleration_power_off_time)
+                ),
+                self.flight.max_acceleration_power_off_time,
             )
+        )
```

Comment:
Sorry for not noticing this before. I might be mistaken, however I believe accelerations expressed in terms of the unit `g` are computed based of the standard gravity `9.80665 m/s²`.

As a reference guide for what I mean, see: this [Wikipedia page](https://en.wikipedia.org/wiki/G-force#Unit_and_measurement).